Makefile

```
@@ -57,6 +57,16 @@ reference-docs: ## Generate reference documentation input YAML files
 	@rm -rf dist/reference
 	@go run internal/documentation/documentation.go -yaml dist/reference
 
+.PHONY: tekton-task-docs
+tekton-task-docs: ## Generate reference documentation input YAML files
+	@rm -rf dist/task
+	@go run internal/task_documentation/task_documentation.go -yaml dist/task
+
+.PHONY: all-docs
+all-docs: ## Generate reference documentation input YAML files
+	@$(MAKE) reference-docs
+	@$(MAKE) tekton-task-docs
```

This is a commonplace pattern of expressing dependencies between targets

```suggestion
all-docs: reference-docs tekton-task-docs ## Generate reference documentation input YAML files
```

---

ack, I had that previously and changed it thinking that we would want to ensure we didn't have failures in the reference creation. I'll use the previous approach.